plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/frontend.ts

```diff
@@ -175,6 +195,54 @@ const addToCartWithOptionsStore = store(
 				);
 				return matchedVariation?.variation_id || null;
 			},
+			get allowsDecrease() {
+				const context = getContext< Context >();
+				const {
+					quantity,
+					childProductId,
+					productType,
+					quantityConstraints,
+					productId,
+				} = context;
+				const currentQuantity =
+					productType === 'grouped' && childProductId
+						? quantity[ childProductId ] || 0
+						: quantity[ productId ] || 0;
+				const id =
+					productType === 'grouped' && childProductId
+						? childProductId
+						: productId;
+				const constraints =
+					quantityConstraints?.[ id ] ||
+					getDefaultConstraints( productType, childProductId );
+				const minValue = constraints.min;
+				const step = constraints.step;
+				return currentQuantity - step >= minValue;
+			},
+			get allowsIncrease() {
+				const context = getContext< Context >();
+				const {
+					quantity,
+					childProductId,
+					productType,
+					quantityConstraints,
+					productId,
+				} = context;
+				const currentQuantity =
+					productType === 'grouped' && childProductId
+						? quantity[ childProductId ] || 0
+						: quantity[ productId ] || 0;
+				const id =
+					productType === 'grouped' && childProductId
+						? childProductId
+						: productId;
```

Comment:
I think this can be simplified like this:

```suggestion
				const id =
					productType === 'grouped' && childProductId
						? childProductId
						: productId;
				const currentQuantity = quantity[ id ] || 0;
```